<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraConnection</class>
 <widget class="QMainWindow" name="CameraConnection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Parker - A Camera Connection Software</string>
  </property>
  <property name="windowIcon">
   <iconset resource="cameraconnectionresources.qrc">
    <normaloff>:/imgs/spider.jpg</normaloff>:/imgs/spider.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>360</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Video</string>
    </property>
    <widget class="QRadioButton" name="rb_video">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>30</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable video mode</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>150</width>
       <height>98</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QPushButton" name="bt_play">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(136, 138, 133);
}</string>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
      <property name="icon">
       <iconset resource="cameraconnectionresources.qrc">
        <normaloff>:/imgs/play.png</normaloff>:/imgs/play.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_stop">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(136, 138, 133);
}</string>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
      <property name="icon">
       <iconset resource="cameraconnectionresources.qrc">
        <normaloff>:/imgs/stop.png</normaloff>:/imgs/stop.png</iconset>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>30</height>
       </size>
      </property>
      <property name="digitCount">
       <number>8</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>170</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Photo</string>
    </property>
    <widget class="QPushButton" name="bt_capture">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(136, 138, 133);
}</string>
     </property>
     <property name="text">
      <string>Capture</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_photo">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable photo mode</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="bt_return">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>90</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(136, 138, 133);
}</string>
    </property>
    <property name="text">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>646</width>
      <height>574</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="bt_connect">
         <property name="minimumSize">
          <size>
           <width>210</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>210</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: rgb(136, 138, 133);
}</string>
         </property>
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bt_disconnect">
         <property name="minimumSize">
          <size>
           <width>210</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>210</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: rgb(136, 138, 133);
}</string>
         </property>
         <property name="text">
          <string>Disconnect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bt_exit">
         <property name="minimumSize">
          <size>
           <width>210</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>210</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: rgb(136, 138, 133);
}</string>
         </property>
         <property name="text">
          <string> Exit</string>
         </property>
         <property name="icon">
          <iconset theme="exit">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="shortcut">
          <string>Ctrl+W</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QComboBox" name="combo_selectDevice">
       <property name="minimumSize">
        <size>
         <width>640</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>640</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox{
	color: rgb(0, 0, 0);
	background-color: rgb(255,255,255);
}</string>
       </property>
       <item>
        <property name="text">
         <string>Select device</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_screen">
       <property name="minimumSize">
        <size>
         <width>640</width>
         <height>480</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>640</width>
         <height>480</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	background-color: rgb(0,0,0);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="cameraconnectionresources.qrc"/>
 </resources>
 <connections/>
</ui>
